meerkat_frontend/src/files/pages/disease.html

```diff
@@ -96,6 +96,8 @@
 	$("#box_heading").html(i18n.gettext('Information'));
 	$("#box_2_heading").html(i18n.gettext('Time of reports'));
 	$("#box_3_heading").html(i18n.gettext('Location of reports'));
+	$("#no-data-available").html(i18n.gettext('There is no data available for this disease'));
+  $("#no-Data-title").html(i18n.gettext('No data available'));
```

Comment:
Probably not a good idea to include a random capital letter here "no-Data-title" to "no-data-title".  Also the indentation is a bit strange on this line.